src/server/maker/maker-worker.ts

```diff
@@ -99,27 +113,16 @@ module Experimental {
             }
 
             export function runMakerFunction(makerCode: string) {
-                let makerFunc: () => any;
+                let result: any;
 
                 // TODO EVENTUALLY: figure out if eval is evil.
```

Comment:
BTW, I think eval is not evil after all, in this context.  So probably OK to remove this todo.